src/Uno.UI.Composition/Composition/CompositionEffectBrush.skia.cs

```diff
@@ -0,0 +1,1726 @@
+					}
+
+					return null;
+				}
+			case IGraphicsEffectD2D1Interop effectInterop:
+				{
+					switch (EffectHelpers.GetEffectType(effectInterop.GetEffectId()))
+					{
+						case EffectType.GaussianBlurEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetPropertyCount() == 3 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									effectInterop.GetNamedPropertyMapping("BlurAmount", out uint sigmaProp, out _);
+									effectInterop.GetNamedPropertyMapping("Optimization", out uint optProp, out _);
+									effectInterop.GetNamedPropertyMapping("BorderMode", out uint borderProp, out _);
+
+									float sigma = (float)(effectInterop.GetProperty(sigmaProp) ?? throw new NullReferenceException("The effect property was null"));
+									_ = (uint?)effectInterop.GetProperty(optProp); // TODO
+									_ = (uint?)effectInterop.GetProperty(borderProp); // TODO
+
+									return SKImageFilter.CreateBlur(sigma, sigma, sourceFilter, new(UseBlurPadding ? bounds with { Left = -100, Top = -100, Right = bounds.Right + 100, Bottom = bounds.Bottom + 100 } : bounds));
+								}
+
+								return null;
+							}
+						case EffectType.GrayscaleEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									return SKImageFilter.CreateColorFilter(
+										SKColorFilter.CreateColorMatrix(
+											new float[] // Grayscale Matrix
+											{
+												0.21f, 0.72f, 0.07f, 0, 0,
+												0.21f, 0.72f, 0.07f, 0, 0,
+												0.21f, 0.72f, 0.07f, 0, 0,
+												0,     0,     0,     1, 0
+											}),
+										sourceFilter, new(bounds));
+								}
+
+								return null;
+							}
+						case EffectType.InvertEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									return SKImageFilter.CreateColorFilter(
+										SKColorFilter.CreateColorMatrix(
+											new float[] // Invert Matrix
+											{
+												-1, 0,  0,  0, 1,
+												0,  -1, 0,  0, 1,
+												0,  0,  -1, 0, 1,
+												0,  0,  0,  1, 0,
+											}),
+										sourceFilter, new(bounds));
+								}
```

Comment:
yeah, I think we should log a warning message here.

> Is it an accepted fallback?
it's not a fallback, this will eventually throw an exception in `UpdatePaint` (because it returned null), this follows the same behavior as Windows